nodejs/README.md

```diff
@@ -0,0 +1,62 @@
+# Stan TV NodeJS Coding Challenge
+
+> Welcome to the Stan TV NodeJS Coding Challenge!
+
+## Overview
+
+The purpose of this challenge is to assess your knowledge around composing a **simple websocket based client/server application** closely following the guidelines and requirements.
+
+This challenge is expected to take around 4-6 hours.
+
+## Guidelines
+
+- The client application should be built using [Babel](https://www.babeljs.io) along with [webpack](https://webpack.js.org) into a `dist` folder containing two files (`logo.svg` and `index.html`).
```

Comment:
I feel like we should add our intent behind using Babel and Webpack here. i.e. need for running the app on the older browsers.

Reply:
from a backend pov not required as you need to do this to run typescript (the alternative is using ts-node but yeah)